laozi-mainnet/docs/prepare-for-upgrade-v2.5.md

```diff
@@ -0,0 +1,47 @@
+# Bandchain: How to prepare for upgrade v2.5
+
+This document describes methods how to prepare for upgrade v2.5
+
+> Note: If you are not running node by Cosmovisor, please follow this guide first (Bandchain: [How to migrate from Bandd binary to Cosmovisor](https://github.com/bandprotocol/launch/blob/master/laozi-mainnet/docs/migrate-bandd-binary-to-cosmovisor.md))
+
+### Step 1: Clone & Install the new BandChain Laozi
+Make new bandd binary from chain v2.5.1
+
+```
+# Clone Bandchain Laozi version 2.5.1
+source ~/.profile
+cd ~
+git clone https://github.com/bandprotocol/chain
+cd chain
+git fetch
+git checkout v2.5.1
+
+# Install binaries to $GOPATH/bin
+make install
+
+# Verify bandd version
+bandd version
```

Comment:
What is expect result after running this command?

Reply:
added.